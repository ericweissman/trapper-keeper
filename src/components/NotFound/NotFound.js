import React from 'react'
import { Link } from 'react-router-dom'

export const NotFound = () => {
  return (
    <div className='not-found'>
      <h3>404 Not Found</h3>
      <Link to='/'>Return to All Notes</Link>
    </div>
  )
}

export default NotFound