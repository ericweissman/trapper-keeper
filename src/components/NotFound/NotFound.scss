.not-found {
  text-align: center;
  font-family: $font-primary;
  h3 {
    color: $primary-dark;
    text-transform: uppercase; 
  }
  a {
    text-decoration: underline;
    color: $primary-light;
  }
  padding-top: 50px;
}