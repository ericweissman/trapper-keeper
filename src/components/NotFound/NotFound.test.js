import React from 'react'
import { shallow } from 'enzyme'
import NotFound from './NotFound'

describe('NotFound', () => {
  let wrapper
  it('should have match the correct snapshot', () => {
    wrapper = shallow(<NotFound />)
    expect(wrapper).toMatchSnapshot()
  })
})