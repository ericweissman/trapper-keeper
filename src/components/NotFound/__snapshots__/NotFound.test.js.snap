// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should have match the correct snapshot 1`] = `
<div>
  <h1>
    404 Not Found
  </h1>
  <Link
    replace={false}
    to="/"
  >
    Return to All Notes
  </Link>
</div>
`;
